frontend/components/FieldColor/FieldColorEdit.jsx

```diff
@@ -0,0 +1,262 @@
+'use strict'
+
+import {h, Component} from 'preact'
+import proptypes from 'proptypes'
+
+import Style from 'lib/Style'
+import styles from './FieldColor.css'
+
+import ColorPicker from 'components/ColorPicker/ColorPicker'
+import Label from 'components/Label/Label'
+import TextInput from 'components/TextInput/TextInput'
+
+/**
+ * Component for API fields of type Color
+ */
+export default class FieldColorEdit extends Component {
```

Comment:
Sorry to be a pain, but do you mind sorting the methods alphabetically (with the exception of the `constructor`, which should go first)? I think it makes things a lot easier to read and find. Some older components don't enforce that, but I've been trying to implement it as we go.